AutoGraph/SubscriptionRequest.swift

```diff
@@ -74,6 +74,7 @@ public enum GraphQLWSProtocol: String {
     case data = "data"                                 // Server -> Client
     case error = "error"                               // Server -> Client
     case complete = "complete"                         // Server -> Client
+    case unknown
```

Comment:
might be an improvement if it included `unknown_received_payload(String)` and we put what we got in into String

Reply:
naming as well, so it's obvious this is a Server -> Client thing

Reply:
wups, in camelCase though `unknownReceivedPayload(String)`

Reply:
or `unknownResponse(String)`

Reply:
So I don't believe you can have associated value with raw enums. I can change the name though